.circleci/config.yml

```diff
@@ -1,6 +1,27 @@
-version: 2
+version: 2.1
```

Comment:
I'd recommend using the CircleCI orbs we have (react-tools and django-tools) to lint and test the project since it helps simplify the CI config. You don't have to move over if you don't want to though. [Clubs has a good example of this](https://github.com/pennlabs/penn-clubs/blob/master/.circleci/config.yml) (you can ignore anything mentioning docker or helm).

Reply:
Right now there's no frontend tests, so we'll move after some frontend tests have been written. I think this is what happened with Clubs as well